images/bind9/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf -y update && \
+    dnf -y install bind bind-utils && \
+    dnf clean all
+
+# set up the BIND env and gen rndc key
+RUN rndc-confgen -a -c /etc/rndc.key && \
```

Comment:
shouldn't this be generated at runtime and maybe mounted as a volume?

Reply:
Not sure!!

Reply:
do you need another suggestion to remove this layer? :)